packages/wrangler/src/queues/cli/config.ts

```diff
@@ -0,0 +1,13 @@
+import path from "path";
+import { type Config, readConfig, findWranglerToml } from "../../config";
+
+export interface Args {
+	config?: string;
+	script?: string;
+}
+
+export function read(args: Args): Config {
```

Comment:
Is there a specific reason this needs special handling? `readConfig` calls `findWranglerToml` internally—usually `readConfig(args.config, args)` should work

Reply:
Ah thanks, no reason for the special handling, removed!